buildandpack.cmd

```diff
@@ -0,0 +1,2 @@
+msbuild .\src\WebAPI.Testing\WebAPI.Testing.csproj /t:Build /p:Configuration="Release 4.5"
```

Comment:
No, its only the final step that needs to package. The package step sweeps all the assemblies together. We could probably break out the package step into its own command call - i did just copy this method from here http://brian-federici.com/blog/2013/4/17/automatically-target-multiple-frameworks-in-your-nuget-package